userspace/sysdig/sysdig.cpp

```diff
@@ -261,9 +261,10 @@ static void usage()
 "                    Stop capturing after <num> events\n"
 " --page-faults      Capture user/kernel major/minor page faults\n"
 " --plugin-config-file\n"
-"                    Use the plugin configuration in a falco-compatible file.\n"
+"                    Load the plugin configuration from a Falco-compatible yaml file.\n"
+"                    Do not mix this option with the '-H' or '-I' options: it is unsupported.\n"
 "                    See the plugin section in https://falco.org/docs/configuration/ for\n"
-"                    additional information\n"
+"                    additional informations\n"
```

Comment:
"information" was correct :D

Reply:
Uh it feels weird in that context to me :D

Reply:
Fixed, you are right of course :)